django/contrib/admin/static/admin/css/widgets.css

```diff
@@ -19,6 +19,7 @@
 
 .selector-chosen select {
     border-top: none;
+    height: 20.8em;
```

Comment:
I think Baptiste's question was how did you calculate it. It looks like it's the sum of `.selector select { height: 17.2em; }` and some other height?

p.s. For clarity, I think this would be better as two separate commits.